### PR TITLE
[KAN-778] Reference list of PRs in images

### PR DESCRIPTION
**Pull request by pertsenga**
## Changes
- Reference PRs contained in build images

## Screenshot
Screenshot 2023-06-02 at 11 26 25 AM

## References
- https://managedapp.atlassian.net/browse/KAN-778

### CODE REVIEW

File: images.js [hunk 1]
```diff
@@ -114,4 +119,37 @@ async function runSkipperListImages(log) {
...
 
+async function getReleases() {
+  const octokit = new Octokit({
+      auth: process.env.GITHUB_TOKEN,
+  })
+  var owner = process.env.GITHUB_ORG;
+  var repo = process.env.GITHUB_REPONAME;
+
+  var resp = await octokit.request(`GET /repos/${owner}/${repo}/releases`);
+  var releases = resp.data.map(release => {
```
[the remark this answers is not in the document]
Comment (pertsenga) [reply images.js]: carried over from copy pasting

File: images.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
 const { exec } = require('child_process')
+const {Octokit} = require("octokit");
 const util = require('util')
+const _ = require('lodash');
```
Comment (acealmazar-managed) [inline images.js:4]: Would prefer to only import specific lodash function used `lodash/chain` etc if said lodash functions are really needed.

File: images.js [hunk 3]
```diff
@@ -1,13 +1,15 @@
 const { exec } = require('child_process')
+const {Octokit} = require("octokit");
```
Comment (acealmazar-managed) [inline images.js:2]:
```suggestion
const { Octokit } = require("octokit");
```

File: images.js [hunk 4]
```diff
@@ -48,7 +50,9 @@ async function Images(command, ack, respond, log) {
 
     let fmt = imgs
       .map((el) => {
-        return `\`${el.split(' ')[1]}\`, hash \`${el.split(' ')[0]}\``
+        const versionName = el.split(' ')[1]
+        const hyperlinks = releases.find(({version}) => version === versionName)?.pullRequestHyperlinks || []
```
Comment (acealmazar-managed) [inline images.js:54]:
```suggestion
        const hyperlinks = releases.find(({ version }) => version === versionName)?.pullRequestHyperlinks || []
```

**Review by acealmazar-managed — CHANGES_REQUESTED** [review]
Nitpick: please be consistent with use of semi-colon. some lines use it while some lines don't

**Comment by pertsenga** [comment]
Thanks @acealmazar-managed will push these changes as a commit. Will consider applying a linter and formatter to this project

**Pull request merged**